coremltools/converters/mil/frontend/torch/test/test_torch_conversion_api.py

```diff
@@ -264,6 +266,65 @@ def test_func_dummy(context, inputs):
         # Cleanup the test
         del _TORCH_OPS_REGISTRY.name_to_func_mapping["test_func_dummy"]
 
+
+@pytest.mark.skipif(not _HAS_TORCH, reason=MSG_TORCH_NOT_FOUND)
+class TestFxNodeSupport:
```

Comment:
Do we have the link of the code from the torch repository which will call this API?
It would be good to include a doc string for this `TestFxNodeSupport` class,
explicitly mentioning that, the API is used by this 3rd party code, so we cannot break it.

Reply:
This API will be used by [ExecuTorch](https://github.com/pytorch/executorch). Specifically, starting from our [partitioner PR](https://github.com/pytorch/executorch/pull/1415)

Should we add doc string to the API function, or to this test class?

Reply:
Added doc string
```
class TestFxNodeSupport:
    """
    The API ``ct.converters.mil.frontend.torch.is_torch_fx_node_supported`` is used
    by 3rd-party code ExecuTorch: https://github.com/pytorch/executorch/pull/1415,
    so we cannot break it
    """

    ...
```